crates/mysticeti-core/src/consensus/base_committer.rs

```diff
@@ -111,11 +120,11 @@ impl BaseCommitter {
             .get_blocks_at_authority_round(leader, leader_round);
 
         // Get all blocks that could be potential supports for the target leader. These blocks
-        // are in the decision round of the target leader and are linked to the anchor.
+        // are in the voting round of the target leader and are linked to the anchor.
         let wave = self.wave_number(leader_round);
-        let decision_round = self.decision_round(wave);
-        let decision_blocks = self.block_store.get_blocks_by_round(decision_round);
-        let potential_supports: Vec<_> = decision_blocks
+        let voting_round = self.voting_round(wave);
```

Comment:
support must be checked at the decision round, not the voting round. 
I mentioned voting round yesterday but the name can be confusing; a better name is perhaps "boosting". It is only there to increases the chances that the decision round will have enough (indirect) support of the leader.